options.html

```diff
@@ -1,23 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Why are we deleting this part of the code? Shouldn't we keep the shell at least, in case we want to add stuff to it later on?